SparkPost/support-docs
# FE-1816 - Fixing getting-started-sparkpost links/media

### What Changed

- Corrected the # link names
- Stripped off `#.*$` from the `asPath` variable in the `getImageSrc` method.
- Added a cypress test to verify images load when the page url has # links present.
- Added a shortcut `test` alias in the package file.
- Added an engine block to specify the working node version.

### How To Test or Verify

- Run the cypress tests
- Manually load the subpages that have images and check that the images load correctly without `#` links in the url
- Manually load the subpages that have images and check that the images load correctly with `#` links in the url
- https://deploy-preview-682--support-docs.netlify.app/docs/getting-started/getting-started-sparkpost/
- https://deploy-preview-682--support-docs.netlify.app/docs/getting-started/getting-started-sparkpost/#signals-analytics

### PR Checklist

Below are some checklists to follow for the correct procedure in different circumstance. The first list ("All PRs Checklist") should be followed for ALL PRs. The next 2 are additive to this list depending on what type of PR you are using.

For example: If you are submitting a content change to one of the support documents, your checklist would include the:

- "All PRs Checklist"
- AND the "Content Changes Checklist

If you are submitting a feature addition, enhancement, or bug fix, your checklist would include the:

- "All PRs Checklist"
- AND the "Development Changes Checklist"

#### All PRs Checklist


#### Content Changes Checklist


#### Development Changes Checklist (some checks are automatic github actions and will not be listed here. ie. "all tests pass")
